IdentityCore/src/broker_operation/response/MSIDBrokerNativeAppOperationResponse.h

```diff
@@ -43,11 +46,21 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, class, readonly) NSNumber *defaultHttpStatusCode;
 @property (nonatomic, nullable) NSDictionary *httpHeaders;
 @property (nonatomic) NSString *httpVersion;
+@property (nonatomic) NSDate *responseGenerationTimeStamp;
+@property (nonatomic) NSDate *requestReceivedTimeStamp;
 
 - (instancetype)initWithDeviceInfo:(MSIDDeviceInfo *)deviceInfo;
 - (instancetype)init NS_UNAVAILABLE;
 + (instancetype)new NS_UNAVAILABLE;
 
+#if !EXCLUDE_FROM_MSALCPP
+
+- (void)trackPerfTelemetryWithLastRequest:(MSIDLastRequestTelemetry *)telemetry
+                         requestStartDate:(NSDate *)requestStartDate
+                                     type:(NSString *)type;
```

Comment:
suggestion: rename 'type' parameter to 'telemetryType'

Reply:
Addressed